examples/cifar10_cnn.py

```diff
@@ -112,8 +113,8 @@
     datagen.fit(x_train)
 
     # Fit the model on the batches generated by datagen.flow().
-    model.fit_generator(datagen.flow(x_train, y_train,
-                                     batch_size=batch_size),
+    model.fit_generator(datagen.flow(x_train, y_train, batch_size=batch_size),
+                        steps_per_epoch=int(math.ceil(x_train.shape[0] / float(batch_size))),
```

Comment:
Could you wrap `float(batch_size))),` to new line.

Reply:
Thanks, I added wrap.

https://github.com/keras-team/keras/pull/13019/commits/2550d0ca44d58027a34a714745855771acc59fe7